### PR TITLE
on_mesh_prefix parsing fix

### PR DESCRIPTION
**Pull request by priyankaCh22**

### CODE REVIEW

File: silk/tools/wpan_table_parser.py [hunk 1]
```diff
@@ -483,11 +483,12 @@ def __init__(self, text):
         # Example of expected text:
         #
         # `\t"fd00:abba:cafe::       prefix_len:64   origin:user     stable:yes flags:0x31`
-        # ` [on-mesh:1 def-route:0 config:0 dhcp:0 slaac:1 pref:1 prio:med] rloc:0x0000"`
+        # ` [on-mesh:1 def-route:0 config:0 dhcp:0 slaac:1 pref:1 nd-dns:0 dp:0 prio:med] rloc:0x0000"`
 
         m = re.match(
             r"\t\"([0-9a-fA-F:]+)\s*prefix_len:(\d+)\s+origin:(\w*)\s+stable:(\w*).* \[" +
-            r"on-mesh:(\d)\s+def-route:(\d)\s+config:(\d)\s+dhcp:(\d)\s+slaac:(\d)\s+pref:(\d)\s+prio:(\w*)\]" +
+            r"on-mesh:(\d)\s+def-route:(\d)\s+config:(\d)\s+dhcp:(\d)\s+slaac:(\d)\s+pref:(\d)\s+"
```
Comment (qilinkang) [inline silk/tools/wpan_table_parser.py]: Missing  ' +' at the end of statement ?

Comment (priyankaCh22) [reply silk/tools/wpan_table_parser.py]: Ack

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by tzm41 — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Comment by priyankaCh22** [comment]
These new parameters were seen to be introduced with latest wpantund(starting from 0.07.01-356-g523bdab)

**Pull request merged**